app/util.py

```diff
@@ -1,11 +1,26 @@
 from app import mongo
 from bson.objectid import ObjectId
+import requests
 
 
 def serialize_doc(doc):
     doc["_id"] = str(doc["_id"])
     return doc
 
+def slack_msg(msg):

```

Comment:
this function doesn't make sense. how do we identify to which user the msg goes?

Reply:
msg will be going on a channel in this case tms channel or i have created a webhook for a directchannel it will be going on that too